src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/ProjectCard.kt

```diff
@@ -1,23 +1,31 @@
 package org.wycliffeassociates.otter.jvm.app.widgets
 
 import javafx.geometry.Pos
+import javafx.scene.control.Button
+import javafx.scene.layout.Pane
 import javafx.scene.layout.VBox
 import org.wycliffeassociates.otter.common.data.model.Project
 import tornadofx.*
 import tornadofx.Stylesheet.Companion.root
 
 class ProjectCard(project: Project) : VBox() {
+    val loadButton = Button("Load")
```

Comment:
the text for this button can't be hardcoded here. we need to localize it

Reply:
Just leave it blank here and let the parent layout set the value?

Reply:
yes! I believe you should be able to do something like 

``` projectpage { 
                             loadButton.apply {
                                text = messages["load"]
                  }
} ```